documentation/source/ObjectDetection.md

```diff
@@ -0,0 +1,312 @@
+# Object Detection
+
+Object detection is a core task in computer vision that allows to detect and classify bounding boxes in images. 
```

Comment:
let's make this intro reacher. maybe with GPT...

Reply:
I'm out of requests in chatGPT, hoping I'm poetic enough without it :D 6e116f9